.flake8

```diff
@@ -0,0 +1,10 @@
+[flake8]
+select = B,B9,C,DAR,E,F,N,RST,S,W
```

Comment:
this excludes the `D` checks (docs) because I hate them for projects that aren't pure apis